cmd/kpod/spec.go

```diff
@@ -154,12 +175,6 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 	}
 
 	/*
-				// Rlimits []PosixRlimit // Where does this come from
-				// Type string
-				// Hard uint64
-				// Limit uint64
-				OOMScoreAdj: &config.resources.oomScoreAdj,
```

Comment:
Maybe keep this? Don't think we have OOM adjust in there yet

Reply:
I had a patch for this.  Seem to have lost it, will add it back for now.